packages/math/texlike.lua

```diff
@@ -479,6 +479,10 @@ compileToMathML(
   \def{sqrt}{\msqrt{#1}}
   \def{bi}{\mi[mathvariant=bold-italic]{#1}}
   \def{dsi}{\mi[mathvariant=double-struck]{#1}}
+  \def{lim}{\mo{lim}}
```

Comment:
Checking Pandoc TeXmath, at a glance, other operators such as min, max, perhaps ker and others should also behavelike this. Ideas?

Reply:
From the amsmath (§5.1) list of predefined operators: 
 arcos,  deg,    lg,     projlim,
 arcsin, det,    lim,    sec,
 arctan, dim,    liminf, sin,
 arg,    exp,    limsup, sinh,
 cos,    gcd,    ln,     sup,
 cosh,   hom,    log,    tan,
 cot,    inf,    max,    tanh,
 coth,   injlim, min,
 csc,    ker,    Pr

The 10 following ones (tested on https://latexeditor.lagrida.com/) have subscripts going under the operator:
projlim, lim, liminf, limsup, gcd, sup, inf, max, injlim, min

For the record, amsmath has 4 special operators varinjlim varliminf varprojlim varlimsup which show as "lim" with stacked long arrows (of the size taken by "lim") or bars. I'll leave these out of the current scope.